process-compose/movement-full-node/process-compose.test-followers.yml

```diff
@@ -16,31 +18,40 @@ processes:
         command: echo "true"
 
   setup-follower-1:
+    environment:
+      - "ETH_RPC_CONNECTION_PROTOCOL=http"
+      - "ETH_RPC_CONNECTION_HOSTNAME=0.0.0.0"
+      - "ETH_RPC_CONNECTION_PORT=8090"
+      - "ETH_WS_CONNECTION_PROTOCOL=ws"
+      - "ETH_WS_CONNECTION_HOSTNAME=0.0.0.0"
+      - "ETH_WS_CONNECTION_PORT=8090"
+      - "MAYBE_RUN_LOCAL=true"
+      - "MOVEMENT_DA_LIGHT_NODE_HTTP1=true"
     command: |
-      sleep 120
+      sleep 30
       export AWS_REGION=us-west-2
       export MAYBE_RUN_LOCAL=false
       export MAYBE_DEPLOY_MCR=false
       export MAPTOS_INDEXER_GRPC_LISTEN_PORT=31734
       export MAPTOS_API_CONNECTION_PORT=31731
       export MAPTOS_API_LISTEN_PORT=31731
       export DOT_MOVEMENT_PATH=$DOT_MOVEMENT_PATH-follower-1
-      export MOVEMENT_SYNC="follower::follower-test-$MOVEMENT_SHARED_RANDOM_1<=>{maptos,maptos-storage,movement-da-db}/**"
+      export MOVEMENT_SYNC="follower::follower-test-$MOVEMENT_SHARED_RANDOM_1<=>{default_signer_address_whitelist,maptos,maptos-storage,movement-da-db}/**"
       movement-full-node-setup
     depends_on:
       build:
         condition: process_completed_successfully
       movement-full-node:
         condition: process_healthy
     readiness_probe:
-      initial_delay_seconds: 180
+      initial_delay_seconds: 40
       exec:
         command: echo "true"
 
   movement-full-follower-1:
     command: |
       export DOT_MOVEMENT_PATH=$DOT_MOVEMENT_PATH-follower-1
-      movement-full-node
+      movement-full-node run
```

Comment:
If `run` is not the default command already, perhaps it should be made one, to preserve behavior in all scripts that were added so far. OTOH, we have recently renamed the binary, so maybe it's time for introducing a mandatory command.